test_requirements.txt

```diff
@@ -0,0 +1,5 @@
+pytest>=5.2.0
```

Comment:
It is better to >= do this as pip dep solver tend to be conservative? otherwise.